# Indicate which sound file to play at what time.
duration time_from_start
string file_name